### ISSUE TITLE
Similar libraries and services

### ISSUE THREAD

**Issue by hrsma2i** [KevinMusgrave/pytorch-metric-learning#7]
I think the frameworks of the following libraries and services are similar to or helpful for this repository.

- [SMORe](https://github.com/cnclabs/smore) : a library to train recommendation models to learn graph data
- [AWS SageMaker Object2Vec](https://aws.amazon.com/jp/blogs/machine-learning/introduction-to-amazon-sagemaker-object2vec/):  a service to easily train metric learning models and easily deploy them as inference API.

FYI.

I'm also interested in metric learning. I'll contribute this repository. I want to connect different domains by metric learning such as face recognition, recommendation, NLP(RNN, word2vec, topic model), image retrieval, multi-modal embedding, extreme classification, few-shot learning. The tasks in these domains are represented as classifications with enormous classes, which are metric learning's strong asset.

**Comment by hrsma2i** [comment]
https://arxiv.org/abs/1905.02479

**Comment by KevinMusgrave** [comment]
Thanks for your interest! Contributions would be greatly appreciated! 

The paper you linked looks good. Can you make a separate issue for implementing it?

**Comment by hrsma2i** [comment]
@KevinMusgrave 
Thank you!
I separated it.
